Modules/mathmodule.c

```diff
@@ -3306,9 +3306,87 @@ math_comb_impl(PyObject *module, PyObject *n, PyObject *k)
 }
 
 
+/*[clinic input]
+math._as_integer_ratio
+    x: object
+    /
+
+Return integer ratio.
+
+Return a pair of integers, whose ratio is exactly equal to the original
```

Comment:
Can we specify that the pair of integers returned are contained within a tuple?

```suggestion
Return a tuple containing a pair of integers, whose ratio is exactly equal to the original
```

Reply:
This was copied from the docstring of `float.as_integer_ratio`. Other `as_integer_ratio` methods have similar wording.

Reply:
This suggestion was based on the docstring Raymond recently created for ``Fraction.as_integer_ratio()``, I made a similar [suggestion](https://github.com/python/cpython/pull/15212#discussion_r312749680) that he added to the PR.

If it would be helpful, I could create a separate PR to make a similar change to ``float.as_integer_ratio()``.

Reply:
The tuple *is* the pair, so I find "tuple containing a pair" a bit confusing. That almost sounds like `((numerator, denominator),)` or something. Why not be explicit and write "a tuple `(numerator, denominator)`"?

Reply:
> Why not be explicit and write "a tuple (numerator, denominator)"?

That would be an improvement. I mostly just wanted to specify that the function returned a tuple.

Reply:
"Pair" is a synonym of 2-element tuple. If you think that this term is incorrect, please open a separate issue and analyze all uses of in in the code and in the documentation (there are a lot of occurrences).

Reply:
> "Pair" is a synonym of 2-element tuple.

Either is probably fine, I just figured it was worth trying to be more technically descriptive. Thanks.